### DISCUSSION TITLE
MOC suggestions

### DISCUSSION THREAD

**Discussion by Lymarian** [NotePlan/plugins#602] [Ideas]
testing v0.3.1
Observations:
The MoC plugin seems to be a sort of search. I added a MoC for "health". My calendar notes have a lot of instances of the tag #health but none of those notes appeared on the MoC results. I then added a new note with the term "health" and that was found by the plug-in. But, the MoC did not return a second instance of "health" in that new note. And, the MoC did not return instances of "health" in additional notes I created. It might be that the Click to Refresh feature was not working.

Suggestions
1. Include Calendar notes in the MoC 'search'.
2. If a calendar note is found, include the surrounding text in the MoC, not just the date of the calendar notes. (Similar to what's displayed by the search function.)
3. Verify that the Click to Refresh feature is working on the plugin.

**Comment by jgclark** [comment]
Thank you for these observations and suggestions.  I will start a new feature request from this.
